tests/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -284,6 +284,9 @@ object SyntaxSuite {
 
     val b = mock[B]
     val fb1: F[B] = fa.as(b)
+
+    val c = mock[F[Boolean]]
+    c.ifF(1,0)
```

Comment:
I bet this is what the formater wants!

```suggestion
    c.ifF(1, 0)
```

Reply:
wow thanks